mapbox_gl_web/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: mapbox_gl_web
 description: Web platform implementation of mapbox_gl
-version: 0.16.0
+version: 0.16.1
```

Comment:
no need to bump the versions

Reply:
Reverted it 👍🏻
But still don't know why the build fails tho